### PR TITLE
Add XCTest for WebViews to UITests target

### PR DESCRIPTION
**Pull request by dsame**
Add xcuitest in order to confirm TextFields implementations of web controllers always provides \U0000fffc readouts

The key line is https://github.com/calabash/DeviceAgent.iOS/blob/fb0d333a724ec4279495d7b5b78ef6a36fb7fa88/TestAppUITests/TestAppUITests.m#L49

### CODE REVIEW

File: TestAppUITests/TestAppUITests.m [hunk 1]
```diff
@@ -0,0 +1,68 @@
+//
+//  TestAppUITests.m
+//  TestAppUITests
+//
+//  Created by  Sergey Dolin on 23/07/2019.
+//  Copyright © 2019 Calabash. All rights reserved.
+//
```
Comment (jmoody) [inline TestAppUITests/TestAppUITests.m]: Remove these comments.

File: TestAppUITests/TestAppUITests.m [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+@implementation TestAppUITests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+
+    // In UI tests it is usually best to stop immediately when a failure occurs.
+    self.continueAfterFailure = YES;
+
+    // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
```
Comment (jmoody) [inline TestAppUITests/TestAppUITests.m]: Remove boilerplate comments.

File: TestAppUITests/TestAppUITests.m (hunk 1 above)
Comment (dsame) [reply TestAppUITests/TestAppUITests.m]: done

Comment (dsame) [reply TestAppUITests/TestAppUITests.m]: UITest target has some problem not easy to fix and can not be build - some files seem to be messed.
Besides i do not think it would be good to mess Calabash and XCTests in the same target: different dependencies, versions, so on. 

Please confirm:
1. i should move XCTest to UITest target and do not fix the build 
2. i should move XCTest to UITest target and try to fix the build
3. keep XCTest and Calabash tests in the separated targets

Comment (max-lobanov) [reply TestAppUITests/TestAppUITests.m]: Honestly, I prefer #2 way.
Please try to pull fresh develop to your branch. Probably, your issues will be fixed

Comment (jmoody) [reply TestAppUITests/TestAppUITests.m]: #324 fixes the build.

File: DeviceAgent.xcworkspace/xcshareddata/xcschemes/TestApp.xcscheme [hunk 3]
```diff
@@ -51,6 +51,16 @@
...
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "4166C9AB22E7009800C8BEBF"
+               BuildableName = "TestAppUITests.xctest"
+               BlueprintName = "TestAppUITests"
+               ReferencedContainer = "container:DeviceAgent.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
```
Comment (jmoody) [inline DeviceAgent.xcworkspace/xcshareddata/xcschemes/TestApp.xcscheme]: This looks like it does not belong in this change set.

File: DeviceAgent.xcworkspace/xcshareddata/xcschemes/UITest.xcscheme [hunk 4]
```diff
@@ -38,6 +38,16 @@
...
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "4166C9AB22E7009800C8BEBF"
+               BuildableName = "TestAppUITests.xctest"
+               BlueprintName = "TestAppUITests"
+               ReferencedContainer = "container:DeviceAgent.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
```
Comment (jmoody) [inline DeviceAgent.xcworkspace/xcshareddata/xcschemes/UITest.xcscheme]: This looks like it does not belong in this change set.

File: TestApp/UITests/XCWebViews.m [hunk 5]
```diff
@@ -0,0 +1,58 @@
+//
+//  TestAppUITests.m
+//  TestAppUITests
+//
```
Comment (jmoody) [inline TestApp/UITests/XCWebViews.m]: Remove code comments.

File: DeviceAgent.xcworkspace/xcshareddata/xcschemes/UITest.xcscheme (hunk 4 above)
Comment (dsame) [reply DeviceAgent.xcworkspace/xcshareddata/xcschemes/UITest.xcscheme]: I cleaned up all references of "TestAppUITests". It is not mentioned anywhere in the PR anymore
